addons/portal/static/src/chatter/portal/thread_model_patch.js

```diff
@@ -0,0 +1,17 @@
+import { Thread } from "@mail/core/common/thread_model";
+
+import { patch } from "@web/core/utils/patch";
+
+const threadPatch = {
+    getFetchParams() {
+        const fetchParams = super.getFetchParams();
+        if (this.model === "discuss.channel" || this.model === "mail.box") {
+            return fetchParams;
+        }
+        return {
+            ...fetchParams,
+            fetch_portal_message: true,
```

Comment:
I'm not a big fan of this name, as portal messages are fetched regardless of the value of this param.

The proper name is more only_portal_message (or only_share_messages as the name of the stuff in python) or ignore_internal_message?

In short, maybe just `from_portal` or `only_portal` or `only_share`?